.github/workflows/ci.yml

```diff
@@ -182,4 +182,91 @@ jobs:
         with:
           name: MacOS CMake Test Logs
           path: build-aux/artifacts
-    
```

Comment:
The newline was deleted at the end of this file. It's just a minor thing, but can you please bring it back?